test/cpp/interop/client.cc

```diff
@@ -222,17 +222,13 @@ int main(int argc, char** argv) {
           factories.emplace_back(
               new grpc::testing::MetadataAndStatusLoggerInterceptorFactory());
         }
-        grpc::ChannelArguments arguments;
         std::string service_config_json =
             absl::GetFlag(FLAGS_service_config_json);
         if (!service_config_json.empty()) {
-          arguments.SetServiceConfigJSON(std::move(service_config_json));
-        }
-        for (const auto& arg : extra_arguments) {
-          arguments.SetPointer(arg.first, arg.second);
+          arguments.SetServiceConfigJSON(service_config_json);
         }
         return CreateChannelForTestCase(test_case, std::move(factories),
-                                        std::move(arguments));
+                                        arguments);
```

Comment:
This should still use `std::move()`.

Reply:
clang-tidy does not allow it...
```
test/cpp/interop/client.cc:231:41: error: passing result of std::move() as a const reference argument; no move will actually happen [performance-move-const-arg,-warnings-as-errors]
                                        std::move(arguments));
                                        ^~~~~~~~~~         ~
Suppressed 9286 warnings (9210 in non-user code, 76 NOLINT).
```

Reply:
Ah, I didn't realize that `CreateChannelForTestCase()` takes the argument as a const reference.  It might make sense to change it to do so, but the way you have it now is fine too.